packages/babel-plugin-emotion/test/fs.test.js

```diff
@@ -32,13 +38,34 @@ describe('babel plugin fs', () => {
       babelrc: false
     })
     expect(fs.existsSync).toBeCalledWith(cssFilename)
+    expect(mkdirp.sync).not.toBeCalledWith()
     expect(touch.sync).toBeCalledWith(cssFilename)
     expect(fs.writeFileSync).toHaveBeenCalled()
     expect(fs.writeFileSync.mock.calls[0][0]).toBe(cssFilename)
```

Comment:
In order to use the fs mock, you need to increment the array index like these.

Reply:
More information. https://facebook.github.io/jest/docs/en/mock-function-api.html#mockfnmockimplementationfn

Reply:
That's a mistake. I wanted to check it wasn't called with anything, because the mkdirp should only be used when the `outputDir` property is provided.